### PR TITLE
VAGOV-6167: Adding new links fields to VAMC.

### PR DESCRIPTION
**Pull request by ethanteague**
## What this does
 - Adds two new link fields to VAMC system content type for subscribing to emergency notifications or email lists
 - Moves old email subscription field out of node form

### CODE REVIEW

File: config/sync/field.field.node.health_care_region_page.field_link_facility_emerg_list.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    - field.storage.node.field_link_facility_emerg_list
+    - node.type.health_care_region_page
+  module:
+    - link
+id: node.health_care_region_page.field_link_facility_emerg_list
+field_name: field_link_facility_emerg_list
+entity_type: node
+bundle: health_care_region_page
+label: 'Emergency updates email list'
+description: ''
```
Comment (swirtSJW) [inline config/sync/field.field.node.health_care_region_page.field_link_facility_emerg_list.yml]: Do we need any helpful UX text here?

File: config/sync/field.field.node.health_care_region_page.field_link_facility_news_list.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    - field.storage.node.field_link_facility_news_list
+    - node.type.health_care_region_page
+  module:
+    - link
+id: node.health_care_region_page.field_link_facility_news_list
+field_name: field_link_facility_news_list
+entity_type: node
+bundle: health_care_region_page
+label: 'News and announcements email list'
+description: ''
```
Comment (swirtSJW) [inline config/sync/field.field.node.health_care_region_page.field_link_facility_news_list.yml]: Helpful UX stuff?

File: config/sync/field.field.node.health_care_region_page.field_link_facility_emerg_list.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/field.field.node.health_care_region_page.field_link_facility_emerg_list.yml]: Updated @swirtSJW :+1:

File: config/sync/field.field.node.health_care_region_page.field_link_facility_news_list.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/field.field.node.health_care_region_page.field_link_facility_news_list.yml]: Updated @swirtSJW :+1:

**Comment by ElijahLynn** [comment]
I manually set this CMS Review Environment to use `VAGOV-5999-op-status-et` then ran `composer va:web:build`. Any push after this will reset this as `composer install` will run again. 

`ssh <username>@devshop.cms.va.gov`
`sudo su - aegir`
`cd ~/projects/cms/pr665/docroot/vendor/va-gov/web/`
`git checkout VAGOV-5999-op-status-et`
`cd ~/projects/cms/pr665`
`composer va:web:build` OR the rebuild trigger in CI ENV is fine too.

**Pull request merged**